dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -117,6 +117,27 @@ def test_frequency_preferences(self):
 
         self.delete_cbsd(cbsd_id)
 
+    def test_creating_cbsd_with_the_same_unique_fields_returns_409(self):
+        builder = CbsdAPIDataBuilder()
+
+        self.when_cbsd_is_created(builder.build_post_data())
+        self.when_cbsd_is_created(builder.build_post_data(), expected_status=HTTPStatus.CONFLICT)
+
+    def test_updating_cbsd_returns_409_when_setting_existing_serial_num(self):
+        builder = CbsdAPIDataBuilder()
+
+        cbsd1_payload = builder.build_post_data()
+        cbsd2_payload = builder.build_post_data()
+        cbsd1_payload["serial_number"] = "foo"
```

Comment:
How about adding this to the builder (`with_serial_number`)?

Reply:
done